src/main/java/com/seals/camplanner/commons/services/BaseServiceImpl.java

```diff
@@ -47,7 +51,12 @@ public T save(T entity) {
      */
     @Override
     public void deleteById(Long id) {
-        this.getRepository().deleteById(id);
+        try {
+            this.getRepository().deleteById(id);
+        } catch (EmptyResultDataAccessException e) {
+            throw new NotFoundException(this.getNotFoundMessage(id));
```

Comment:
We could add a `log.debug`line here that logs the original exception. Otherwise it could turn be harder to solve problems here in the future without having the original exception.

Reply:
Good one. It should be ok now.